src/panels.cpp

```diff
@@ -1642,7 +1669,8 @@ void draw_weapon_classic( const player &u, const catacurses::window &w )
     werase( w );
 
     mvwprintz( w, 0, 0, c_light_gray, _( "Weapon  :" ) );
-    mvwprintz( w, 0, 10, c_light_gray, utf8_truncate( u.weapname(), getmaxx( w ) - 24 ) );
+    nc_color color = c_light_gray;
+    print_colored_text( w, 0, 10, c_light_gray, color, u.weapname( getmaxx( w ) - 24 ) );
```

Comment:
And everywhere else, where `color = c_light_gray`.
```suggestion
    print_colored_text( w, 0, 10, color, color, u.weapname( getmaxx( w ) - 24 ) );
```